<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vista_nuovo_dipendente</class>
 <widget class="QWidget" name="vista_nuovo_dipendente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuovo Dipendente</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_nome">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cognome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_cognome">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Codice fiscale</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_cf">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Indirizzo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_indirizzo">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_email">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_telefono">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data di nascita (dd/MM/yyyy)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_datanascita">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Luogo di nascita</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_luogonascita">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Codice licenza</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_licenza">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bottone_annulla">
        <property name="text">
         <string>Annulla</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bottone_conferma">
        <property name="text">
         <string>Conferma</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
